### PR TITLE
Added TIC-80 Runner

### PR DESCRIPTION
**Pull request by telanus**

### CODE REVIEW

File: lutris/runners/tic80.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        {
+            'option': 'skip',
+            'type': 'bool',
+            'label': 'Skip startup animation',
+            'default': False
+        },
+        {
+            'option': 'sound',
+            'type': 'bool',
+            'label': 'Start in silent mode',
```
Comment (RobLoach) [inline lutris/runners/tic80.py]: Rename to "Mute"?

File: lutris/runners/tic80.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            'default': False
+        },
+        {
+            'option': 'skip',
+            'type': 'bool',
+            'label': 'Skip startup animation',
+            'default': False
+        },
+        {
+            'option': 'sound',
```
Comment (RobLoach) [inline lutris/runners/tic80.py]: Having the option key match the argument makes the code easier to read.... Rename it to `nosound`?

File: lutris/runners/tic80.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    ]
+
+    def play(self):
+        """Run the game."""
+        arguments = [self.get_executable()]
+        if self.runner_config.get('surf'):
+            arguments.append('-surf')
+        if self.runner_config.get('skip'):
+            arguments.append('-skip')
+        if self.runner_config.get('sound'):
```
Comment (RobLoach) [inline lutris/runners/tic80.py]: If you rename the option to `nosound`, the logic will be easier to grasp.

File: lutris/runners/tic80.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            'type': 'bool',
+            'label': 'Start in silent mode',
+            'default': False
+        }
+    ]
+
+    def play(self):
+        """Run the game."""
+        arguments = [self.get_executable()]
+        if self.runner_config.get('surf'):
```
Comment (RobLoach) [inline lutris/runners/tic80.py]: I see a `-fullscreen` option we could use here too! https://github.com/nesbox/TIC-80/wiki#command-line-arguments

**Comment by RobLoach** [comment]
This looks great! Added a few notes of how I think it could improve :+1:

File: lutris/runners/tic80.py (hunk 4 above)
Comment (telanus) [reply lutris/runners/tic80.py]: Fullscreen doesn't seem to work. I created a issue on their [github](https://github.com/nesbox/TIC-80/issues/626), will add the option as soon as it works

**Pull request merged**